pkgs/tools/networking/ddclient/default.nix

```
@@ -38,9 +38,9 @@ perlPackages.buildPerlPackage rec {
 
   meta = with lib; {
     description = "Client for updating dynamic DNS service entries";
-    homepage    = "https://sourceforge.net/p/ddclient/wiki/Home/";
-    license     = licenses.gpl2Plus;
+    homepage = "https://ddclient.net/";
+    license = licenses.gpl2Plus;
     # Mostly since `iproute` is Linux only.
-    platforms   = platforms.linux;
+    platforms = platforms.linux;
```

```suggestion
    platforms = platforms.linux;
    maintainers = with maintainers; [ SuperSandro2000 ];
```

I am one of the upstream devs but currently a bit inactive. Nevertheless.